From 308ac6d0db0fb1ebd73a77600cb2b3c9509090c1 Mon Sep 17 00:00:00 2001
From: Elektroschmock <elektroschmock78@googlemail.com>
Date: Tue, 26 Jun 2018 22:04:57 +0200
Subject: [PATCH 016/296] hwui: Remove deprecated compiler flag

 * HWUI_COMPILE_FOR_PERF uses the -mapcs flag
   but it has already been deprecated.
 * Remove the flag "-mapcs" completely
---
 libs/hwui/Android.bp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libs/hwui/Android.bp b/libs/hwui/Android.bp
index 0bec42733de..c2c9367771f 100644
--- a/libs/hwui/Android.bp
+++ b/libs/hwui/Android.bp
@@ -4,7 +4,7 @@ cc_defaults {
         "hwui_static_deps",
 
         //"hwui_bugreport_font_cache_usage",
-        //"hwui_compile_for_perf",
+        "hwui_compile_for_perf",
     ],
 
     cflags: [
@@ -84,7 +84,6 @@ cc_defaults {
     cflags: [
         "-fno-omit-frame-pointer",
         "-marm",
-        "-mapcs",
     ],
 }
 
-- 
2.11.0

